### ISSUE TITLE
quest=14234

### ISSUE THREAD

**Issue by mygithome002** [jubel02/NomadicWow#2]
**Description**:quest=14234 bug

**Current behaviour**: 

**Expected behaviour**: 

**Steps to reproduce the problem**:

1. http://www.wowhead.com/quest=14234 "The Enemy of My Enemy" :Aggra(only gm mode visible) cannot dialogue,quest cannot complete，Subsequent quests are broken。 i try updating the DB creature_template ,still not work
2. 
3. 

**ArkCore commit / Repack core & DB version**:  0216

**Comment by jubel02** [comment]
So you were lazy enough to just copy paste your comment from Arkania. I see, i see.
I was working on Worgen area, Frank was working on Goblin area. If you could explain very well your problem on Arkania repo you might get good answer. Also I'll check by myself tomorrow or day after. I'm so busy recently.

**Comment by mygithome002** [comment]
i am sorry;I tested a lot of 434 version, see NPC does not speak, so I changed some of the NPC data, mentoring so that he can communicate, and finally, or failed

**Comment by jubel02** [comment]
Yes Arkania ng_world file will give errors on default MySQL server. It needs modification, i told him but seems he doesn't care. But I will upload clean world version soon.

I guarantee you will NOT find any other open source 4.3.4 core better than this :) 
make your choice wisely

**Comment by mygithome002** [comment]
Thank you for your detailed answer, I just like arkcore, so come here to ask questions.

**Comment by mygithome002** [comment]
By the inspiration of your word, I canceled the "phaseid" and "phasegroup" inside core, and compile successful ;
 from the original database to extract the appropriate project(DB creature and DB gameobject), into the new database, the world server can not start, display can not connect mysql

**Comment by jubel02** [comment]
I saw your edited message. I do highly recommend DO NOT mess with database files if you have no idea what you're doing. I uploaded world database with latest update.

Instead of messing with files, I suggest you to do the proper feedback.
_Example:_ **Goblin starting area feedback**
Played character: Goblin Mage _(This need to mentioned, because of phase and flags, some class may have bugged, some are not.)_
Goblin starting area quest line:
1. wowhead.com/quest=1 (working)
2. wowhead.com/quest=2 (working)
3. wowhead.com/quest=3 (working)
4. wowhead.com/quest=4 (working)
5. wowhead.com/quest=5 (working)
6. wowhead.com/quest=6 (not working)
**Bug:** Quest starter NPC is not visible for players, but GM.
7. wowhead.com/quest=7 (didn't test because of didn't complete previous quest)
8. wowhead.com/quest=8 (didn't test because of didn't complete previous quest)
9. wowhead.com/quest=9 (didn't test because of didn't complete previous quest)
10. wowhead.com/quest=10 (didn't test because of didn't complete previous quest)

**Comment by mygithome002** [comment]
Enthusiastic man ! i am going to teat again ang give some feedback.

**Comment by jubel02** [comment]
Don't forget to download latest clean database and compile latest core.

**Comment by phulo** [comment]
in arkore code works around 50 quest of golbin zone, a minor bug in the lost isle in the cave escorting the ore car and take the pictures, but all ok until the pride of kezan quest.
